cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ReplicationController
 metadata: 
   name: monitoring-heapster-v4
-  namespace: default
+  namespace: system
```

Comment:
I don't care much, but I prefer descriptive names.  I agree there could be namespace clashes. 'k8s-system'?